MvvmCross/Platforms/Android/Views/Fragments/MvxBindingFragmentAdapter.cs

```diff
@@ -111,6 +111,17 @@ protected override void HandleCreateViewCalled(
             object? sender, MvxValueEventArgs<MvxCreateViewParameters> e) =>
             FragmentView?.EnsureBindingContextIsSet(e.Value.Inflater);
 
+        protected override void HandleResumeCalled(object? sender, EventArgs e)
+        {
+            IMvxMultipleViewModelCache? cache = null;
+            if (Mvx.IoCProvider?.TryResolve(out cache) == true && FragmentView?.ViewModel != null)
+            {
+                // clear cache if still there
+                cache!.GetAndClear(FragmentView.ViewModel.GetType(), FragmentView.UniqueImmutableCacheTag);
+                return;
+            }
+        }
+
```

Comment:
The `IMvxMultipleViewModelCache` field can be inlined

```suggestion
        protected override void HandleResumeCalled(object? sender, EventArgs e)
        {
            if (Mvx.IoCProvider?.TryResolve(out IMvxMultipleViewModelCache? cache) == true && cache != null && FragmentView?.ViewModel != null)
            {
                // clear cache if still there
                cache.GetAndClear(FragmentView.ViewModel.GetType(), FragmentView.UniqueImmutableCacheTag);
                return;
            }
        }

```

Reply:
Thanks for the feedback. I have applied the change.